vetiver/server.py

```diff
@@ -137,9 +114,7 @@ async def rapidoc():
 
         return app
 
-    def vetiver_post(
-        self, endpoint_fx: Callable, endpoint_name: str = "custom_endpoint"
-    ):
+    def vetiver_post(self, endpoint_fx: Callable, endpoint_name: str = None, **kw):
```

Comment:
Does the `endpoint_name` help the caller? My 2 cents, we can just derive it from `endpoint_fx` as it's internal to the server code.

Reply:
Ah ignore this, got confused with something else I was looking at